_posts/2021-09-07-emergent-practices.md

```diff
@@ -0,0 +1,96 @@
+
+## Make invisible labor visible, and valued
+We discussed how enabling people to secure a livelihood with their project does not just create more resiliency, it can be a way to make visible the types of the undervalued labor that too often goes undercompensated. Especially with tech-focused projects, all the work that goes into ensuring the tool or platform is successful is usually not as rewarded as the work of building the tool itself. That includes writing documentation, onboarding new contributors and end-users, and publicizing the product. A radical approach to money can get us to the root of these dynamics, and brings these all-to-prevalent labor inequities to the fore.
+
+## Addressing the accessibility of tech tools
+Across the board, there was an agreement that many of the tools we build and use remain inaccessible to many. Scott acknowledged how there’s still a lot of work to be done in the cryptocurrency ecosystem, as well as in open source software where major technical contributors are prioritized over individual members of the community. We then discussed how with greater cooperation, tooling, and patterns of working together, that the tech side of projects could just be another specialization, as a role that could be designated to someone. This was part of a greater recognition that we need more diversity within projects to specialize in different aspects of an organization. Tools like Open Collective and organizations like Hypha Co-operative help remove the financial, legal, and administrative barriers for those trying to build their project or address frontline needs, by enabling them to outsource this work.
+
+~
+
+The strength of decentralized systems comes from the interdependence of its parts. This is especially true when it comes to the open and interoperable Web, but applies to any resilient social ecosystem. For groups to be both decentralized and interdependent, they need to be trust-full and honest about how they in fact practice what they preach. Financial transparency is one important place where this happens — to be open and communicative about where and how projects receive money, and how they are compensating people for their labor.
+
+The projects highlighted here are experimenting with this type of radical, pragmatic transparency. They’re the kinds of tools that will help people break out of the dominant economic logic that continues to perpetuate social inequities. I’m excited to see how these tools for economic solidarity will grow and lead to even more experimentation. Eventually, they may help us establish radically better practices on creating organizations that are truly, organizationally, decentralized and interdependent.
+
+*** 
+
+## Additional Reading
+* [COMPOST Magazine][compost-mag]
+* [Hypha Organizational Handbook][hypha-handbook]
+* [DisCO Elements][disco-elements]
+* [Emergent Strategies by adrienne maree brown][emergent-strategy]
+* [Decentralized Web Principles][dweb-principles]
```

Comment:
Something wrong with how this renders, I see a paragraph here: https://staging.hypha.coop/dripline/emergent-practices/ can you re-flow this list to see if it resolves?

Reply:
see latest